kale/codegen/generate_code.py

```diff
@@ -32,15 +32,15 @@ def get_volume_parameters(volumes):
             continue
 
         if v['type'] == 'pvc':
-            par_name = f"vol_{v['mount_point'].replace('/', '_').strip('_')}"
+            par_name = "vol_{}".format(v['mount_point'].replace('/', '_').strip('_'))
```

Comment:
let's not break the 80 chars limit:

```suggestion
            mount_point = v['mount_point'].replace('/', '_').strip('_')
            par_name = "vol_{}".format(mount_point)
```